lib/array.js

```diff
@@ -148,11 +148,11 @@ module.exports = (api) => {
         done(err);
       } else {
         count++;
-        if (count === len) done();
+        if (count === len) done(null);
       }
     };
 
-    if (len < 1) return done();
+    if (len < 1) return done(null);
```

Comment:
@tshemsedinov my opinion is base only on face that I want as user want to know what I get in callback and not quantum unstable null/undefined. 
But I've not find any suggestions or specifications from node or v8 developers.
I found [two](http://fredkschott.com/post/2014/03/understanding-error-first-callbacks-in-node-js/) [articles](https://docs.nodejitsu.com/articles/errors/what-are-the-error-conventions/) which says that must be `null` but I think it's not weighty argument